Parsers/ASimFileEvents/README.md

```diff
@@ -0,0 +1,28 @@
+# Azure Sentinel Information Model (ASIM) FileEvent parsers 
+
+This template deploys all ASIM FileEvent parsers. The template is part of the Azure Sentinel Information Mode (ASIM).
+
+The Azure Sentinel Information Mode (ASIM) enables you to use and create source-agnostic content, simplifying your analysis of the data in your Azure Sentinel workspace.
+
+For more information, see:
+
+- [Normalization and the Azure Sentinel Information Model (ASIM)](https://aka.ms/AzSentinelNormalization)
+- [Azure Sentinel FileEvent normalization schema reference](https://aka.ms/AzSentinelFileEventDoc)
+
+<br>
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://aka.ms/AzSentinelFileEventARM)
+
+<br>
+
+The template deploys the following:
```

Comment:
I would rather list here the products and not the files.

Reply:
makes sense. changed text